cli/cfncluster/config_sanity.py

```diff
@@ -33,6 +33,35 @@ def check_resource(region, aws_access_key_id, aws_secret_access_key, resource_ty
         except ClientError as e:
             print('Config sanity error: %s' % e.response.get('Error').get('Message'))
             sys.exit(1)
+    if resource_type == 'EC2IAMRoleName':
+        try:
+            iam = boto3.client('iam', region_name=region,
+                                aws_access_key_id=aws_access_key_id,
+                                aws_secret_access_key=aws_secret_access_key)
+
+            arn = iam.get_role(RoleName=resource_value).get('Role').get('Arn')
+            accountid = boto3.client('sts').get_caller_identity().get('Account')
```

Comment:
Make sure you add get_caller_identity permission in CfnclusterUserPolicy

Reply:
will update docs